src/app/extensions/seo/guards/meta.guard.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, CanActivate, CanActivateChild, RouterStateSnapshot } from '@angular/router';
+import { Store } from '@ngrx/store';
+
+import { SeoAttributes } from 'ish-core/models/seo-attributes/seo-attributes.model';
+
+import { SetSeoAttributes } from '../store/seo';
+
+const defaults: SeoAttributes = {
+  title: 'seo.defaults.title',
+  description: 'seo.defaults.description',
+  robots: 'index, follow',
+  'og:type': 'website',
+};
+
+@Injectable({ providedIn: 'root' })
+export class MetaGuard implements CanActivate, CanActivateChild {
+  constructor(private store: Store<{}>) {}
+
+  canActivate(route: ActivatedRouteSnapshot, _: RouterStateSnapshot): boolean {
```

Comment:
is this testable?

Reply:
Yes, but then the jest test would have to replicate all the routing and meta definitions. I guess it is enough to have the cypress tests here.